src/EPR.Payment.Service.Common.Data/Repositories/PaymentsRepository.cs

```diff
@@ -0,0 +1,66 @@
+﻿using EPR.Payment.Service.Common.Constants;
+using EPR.Payment.Service.Common.Data.Interfaces.Repositories;
+using Microsoft.EntityFrameworkCore;
+
+namespace EPR.Payment.Service.Common.Data.Repositories
+{
+    public class PaymentsRepository : IPaymentsRepository
+    {
+        private readonly AppDbContext _dataContext;
```

Comment:
Replace with the file I had sent. There should be no concrete dependencies. Instead the dependency should be injected in the constructor using an interface.

Reply:
resolved